app/views/components/filter-layout/index.html

```diff
@@ -0,0 +1,355 @@
+{% extends "layouts/base.html" %}
+
+{% block body %}
+
+  {{ govukBackLink({
+    text: "Back",
+    href: "../"
+  }) }}
+
+  <h1 class="govuk-heading-xl">
+    <span class="govuk-caption-xl">Pattern</span> Filter
+  </h1>
+
+      {% set filterOptionsHtml %}
+
+        {{ govukCheckboxes({
+          idPrefix: 'type',
+          name: 'type',
+          classes: 'govuk-checkboxes--small',
+          fieldset: {
+            legend: {
+              text: 'Type',
+              classes: 'govuk-fieldset__legend--m'
+            }
+          },
+          items: [
+            {
+              value: '1',
+              text: 'Divorce (18)',
+              checked: true
+            },
+            {
+              value: '2',
+              text: 'Financial Remedy (5)',
+              checked: true
+            },
+            {
+              value: '3',
+              text: 'Probate (2)'
+            }
+          ]
+        }) }}
+
+        {{ govukCheckboxes({
+          idPrefix: 'status',
+          name: 'status',
+          classes: 'govuk-checkboxes--small',
+          fieldset: {
+            legend: {
+              text: 'Status',
+              classes: 'govuk-fieldset__legend--m'
+            }
+          },
+          items: [
+            {
+              value: '1',
+              text: 'Application drafted',
+              checked: true
+            },
+            {
+              value: '2',
+              text: 'Awaiting payment'
+            },
+            {
+              value: 'magistrate',
+              text: 'Petition paid',
+              checked: false
+            }
+          ]
+          }) }}
+
+        {% endset %}
+
+
+        <div class="moj-filter-layout">
+
+          <div class="moj-filter-layout__filter">
+
+            {{ mojFilter({
+              heading: {
+              text: 'Filter'
+              },
+              selectedFilters: {
+
+              heading: {
+                text: 'Selected filters'
+              },
+
+              clearLink: {
+                text: 'Clear filters'
+              },
+
+              categories: [
+                {
+                  heading: {
+                    text: 'Type'
+                  },
+                  items: [{
+                    href: '/path/to/remove/this',
```

Comment:
It's worth replacing `/path/to/remove/this` with a simple `#` so that the app doesn't error.